build/dockerfiles/entrypoint.sh

```diff
@@ -323,9 +323,14 @@ set_truststore_system_variables() {
 }
 
 add_che_cert_to_truststore() {
+  # Deprecated, CHE_SELF__SIGNED__CERT environment variable is not used after 7.105.0
   if [ "${CHE_SELF__SIGNED__CERT}" != "" ]; then
     add_cert_to_truststore "${CHE_SELF__SIGNED__CERT}" "HOSTDOMAIN"
   fi
+  CERT_PATH="/self-signed-cert/ca.crt"
```

Comment:
What if there are both variable `${CHE_SELF__SIGNED__CERT}` and `/self-signed-cert/ca.crt` existed, and they are intersecting?

Which priority should be?

Reply:
>What if there are both variable ${CHE_SELF__SIGNED__CERT} and /self-signed-cert/ca.crt existed, and they are intersecting?
>
>Which priority should be?

This is an unrealistic scenario, user is not allowed to add or modify env variables.
>If is deprecated, than does it make sense to change the logic?

I prefer to keep the logic with separate if conditions because the env variable is deprecated, so it will be easier to remove the deprecated condition. In your snippet it would require to modify the actual condition.

Reply:
Okay, thanks for the answer!